### PR TITLE
Docs for app collaborators

### PR DESCRIPTION
**Pull request by lostintangent**
This adds the README updates for the new app collaborator feature. I still need to update some screenshots (e.g. to show the new command in the CLI list), but I can do that as a follow up.

### CODE REVIEW

File: cli/README.md [hunk 1]
```diff
@@ -299,4 +346,6 @@ The history will display all attributes about each release (e.g. label, mandator
 
 Additionally, the history displays the install metrics for each release. You can view the details about how to interpret the metric data in the documentation for the `deployment ls` command above.
 
+By default, the history doesn't display the author of each release, but if you are collaborating on an app with other developers, and want to view who released each update, you can pass the additional `--displayAuthor` (or `-a`) flag to the history command.
```
Comment (lostintangent) [inline cli/README.md]: Note that this doc assumes we will go with the name `--displayAuthor` as I proposed in the other PR. If we stick with `--displayReleasedBy` (ugh!), I can change this to reflect that.

File: cli/README.md [hunk 2]
```diff
@@ -111,6 +111,53 @@ you can run the following command:
...
+1. View the app, its collaborators, [deployments](#deployment-management) and [release history](#viewing-release-history) 
+1. [Release](#releasing-app-updates) updates to any of the app's deployments
+1. [Promote](#promoting-updates-across-deployments) an update between any of the app's deployments
+1. [Rollback](#rolling-back-undesired-updates) any of the app's deployments
+
+Inversely, that means that an app collaborator cannot do any of the following:
+
+1. Rename or delete the app
+1. Create, rename or delete new deployments within the app
+1. Add or remove collaborators from the app (*)
```
Comment (dtivel) [inline cli/README.md]: For completeness, also add?

    Transfer app ownership

File: cli/README.md [hunk 3]
```diff
@@ -111,6 +111,53 @@ you can run the following command:
...
+
+Finally, if at some point, you (as the app owner) will no longer be working on the app, and you want to transfer it to another developer (or a client), you can run the following command:
+
+```shell
+code-push app transfer <appName> <newOwnerEmail>
+```
+
+*NOTE: Just like with the `code-push collaborator add` command, this expects that the new owner has already registered with CodePush using the specified e-mail address.*
+
+Once confirmed, the specified developer becomes the app's owner and immediately receives the permissions associated with that role. Besides the transfer of ownership, nothing else about the app is modified (e.g. deployments, release history, collaborators). This means that you will still be a collaborator of the app, and therefore, if you want to remove yourself, you simply need to run the `code-push collaborator rm` command after successfully transfering ownership.
```
Comment (dtivel) [inline cli/README.md]: transferring

**Comment by dtivel** [comment]
Looks good

File: cli/README.md (hunk 2 above)
Comment (lostintangent) [inline cli/README.md]: Good catch! I'll add that.

**Comment by shishirx34** [comment]
LGTM!

**Pull request merged**